apps/charterafrica/src/payload/collections/Organisations.js

```diff
@@ -103,6 +103,18 @@ const Organisations = {
             pt: "Organização sem fins lucrativos",
           },
         },
+        {
+          value: "Political party",
```

Comment:
Do we need these hard-coded i.e. aren't we pulling them from Airtable?

Reply:
We had this as select maybe we convert it to text to avoid invalid type errors

Reply:
we can leave it for now (lets add an issue to track it), but ideally all of these should be editable via Airtable without requiring any code update from us.